From: Sophie Brun <sophie@offensive-security.com>
Date: Wed, 20 Apr 2022 11:17:17 +0200
Subject: Add an explicit message about missing database

Last-Update: 2019-08-19

See issue 5129: add an explicit message to inform the user to run the
embedded script.
Last-Update: 2019-08-19
---
 manipulators/core/mmain.c | 2 +-
 xplico.c                  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/manipulators/core/mmain.c b/manipulators/core/mmain.c
index 650a589..d40b92e 100644
--- a/manipulators/core/mmain.c
+++ b/manipulators/core/mmain.c
@@ -546,7 +546,7 @@ int main(int argc, char *argv[])
 
     /* Geo Ip localization init */
     if (GeoIPLocInit(config_file) == -1) {
-        printf("Download GeoLiteCity.dat from http://geolite.maxmind.com/download/geoip/database/ and gunzip\n");
+        printf("Run /opt/xplico/geolite_update.sh to download GeoLiteCity.dat from http://geolite.maxmind.com/download/geoip/database/ and to gunzip\n");
         sleep(2);
     }
     else {
diff --git a/xplico.c b/xplico.c
index 729bb2d..5a97430 100644
--- a/xplico.c
+++ b/xplico.c
@@ -399,7 +399,7 @@ int main(int argc, char *argv[])
 
     /* Geo Ip localization init */
     if (GeoIPLocInit(config_file) == -1) {
-        printf("Download GeoLiteCity.dat from http://geolite.maxmind.com/download/geoip/database/ and gunzip and see config files\n");
+        printf("Run /opt/xplico/geolite_update.sh to download GeoLiteCity.dat from http://geolite.maxmind.com/download/geoip/database/ and to gunzip. See config files\n");
         if (!(help || capt == FALSE))
             sleep(2);
     }
